content/learn/book/contributing/docs/_index.md

```diff
@@ -33,23 +33,23 @@ A local server should start and you should be able to access a local version of
 We made an extension to the markdown syntax that makes linking to Rust API docs nicer. It also gives the links special formatting. Here are some examples:
 
 * Full Type Path: {{rust_type(type="struct" crate="std", mod="collections", name="HashMap")}}
-    
+
     ```{{curly_open()}}{rust_type(type="struct" crate="std" mod="collections" name="HashMap")}{{curly_close()}}```
 * Short Type: {{rust_type(type="struct", crate="std" mod="collections", name="HashMap", no_mod=true)}}
-    
+
     ```{{curly_open()}}{rust_type(type="struct" crate="std" mod="collections" name="HashMap" no_mod=true)}{{curly_close()}}```
 * Plural Type: {{rust_type(type="struct" crate="std" mod="collections" name="HashMap" no_mod=true plural=true)}}
-    
+
     ```{{curly_open()}}{rust_type(type="struct" crate="std" mod="collections" name="HashMap" no_mod=true, plural=true)}{{curly_close()}}```
 * Function: {{rust_type(type="struct" crate="std" mod="collections" name="HashMap" no_mod=true method="insert")}}
-    
+
     ```{{curly_open()}}{rust_type(type="struct" crate="std" mod="collections" name="HashMap" no_mod=true method="insert")}{{curly_close()}}```
 * Module: {{rust_mod(crate="std" mod="collections")}}
-    
+
```

Comment:
My editor removed trailing whitespace and added a line break to the end of files, if this is undesired I can revert the changes.

Reply:
Seems preferred to me.